src/main/java/org/home/implementation/SimpleMessageStorageImpl.java

```diff
@@ -0,0 +1,46 @@
+package org.home.implementation;
+
+import org.home.interfaces.MessageStorageInterface;
+import org.home.model.Message;
+import org.home.model.MessageQueue;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+
+@Component("primary")
+public class SimpleMessageStorageImpl implements MessageStorageInterface {
+   // MessageQueue messageQueue;
+    Queue<Message> messages;
```

Comment:
is it threadSafe ?
Multiple concurrent request will updating this data structure